src/patternfly/components/ClipboardCopy/docs/code.md

```diff
@@ -0,0 +1,28 @@
+## Overview
+
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-label="[button label text]"` | `.pf-c-clipboard-copy__group-toggle` `.pf-c-clipboard-copy__group-copy` |  Provides an accessible name for the button when an icon is used instead of text. **Required when an icon is used with no supporting text.** |
```

Comment:
```suggestion
| `aria-label="[button label text]"` | `.pf-c-clipboard-copy__group-toggle`, `.pf-c-clipboard-copy__group-copy` |  Provides an accessible name for the button when an icon is used instead of text. **Required when an icon is used with no supporting text.** |
```